### ISSUE TITLE
Function.extractParams is deprecated.

### ISSUE THREAD

**Issue by swsnr** [mmk2410/atom-oss-license#2]
Assigning custom properties to a marker when creating/copying it is
deprecated. Please, consider storing the custom properties you need in
some other object in your package, keyed by the marker's id property.
```
Function.extractParams (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/marker.js:43:14)
MarkerLayer.markPosition (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/marker-layer.js:223:24)
DisplayMarkerLayer.markBufferPosition (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/display-marker-layer.js:109:52)
TextEditor.markBufferPosition (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:1746:38)
Object.restoreCursor (/Users/swiesner/.atom/packages/atom-oss-license/lib/atom-oss-license.coffee:468:23)
Object.addLicense (/Users/swiesner/.atom/packages/atom-oss-license/lib/atom-oss-license.coffee:491:21)
```

**Comment by fjpalacios** [comment]
Same here.

**Comment by mmk2410** [comment]
Let us see what happens in https://github.com/atom/atom/issues/15646

**Comment by mmk2410** [comment]
This will be fixed in atom 1.22

**Comment by saadq** [comment]
I still seem to be getting this issue.

```
Calls to deprecated functions
-----------------------------
logDeprecations @ <embedded>:17967
<embedded>:17970 (1) Function.extractParams : Assigning custom properties to a marker when creating/copying it is
deprecated. Please, consider storing the custom properties you need in
some other object in your package, keyed by the marker's id property. Deprecation
```

<img width="1126" alt="screen shot 2018-02-03 at 10 19 38 pm" src="https://user-images.githubusercontent.com/5678694/35773991-617b8aa2-0930-11e8-9790-bbf11969af90.png">

`atom-oss-license` version 1.1.2

Atom version 1.23.3
